### PR TITLE
Implemented bookmarks global app

### PR DESCRIPTION
**Pull request by iTweeno**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,33 +13,37 @@
     "test:coverage": "cross-env NODE_OPTIONS=--max-old-space-size=1024 jest --maxWorkers=75% --coverage"
   },
   "dependencies": {
-    "@deskpro/app-sdk": "1.2.8",
+    "@deskpro/app-sdk": "1.2.11",
+    "@deskpro/deskpro-ui": "^6.3.1",
```
[the remark this answers is not in the document]
Comment (iTweeno) [reply package.json:17]: ill just remove it

**Review by zpawn — APPROVED** [review]

File: manifest.json [hunk 2]
```diff
@@ -22,5 +22,16 @@
       "isRequired": true,
       "isBackendOnly": false
     }
+  },
+  "proxy": {
+    "whitelist": [
+      {
+        "url": ".*",
```
Comment (AshleyDawson) [inline manifest.json:29]: Why is a completely open proxy required here?

Comment (iTweeno) [reply manifest.json:29]: This was needed to get the favicons, just forgot to change

Comment (iTweeno) [reply manifest.json]: Update works just fine
image

File: manifest.json [hunk 3]
```diff
@@ -22,5 +22,16 @@
       "isRequired": true,
       "isBackendOnly": false
     }
+  },
+  "proxy": {
+    "whitelist": [
+      {
+        "url": "https://www.google.com/s2/favicons?domain=.*",
```
Comment (AshleyDawson) [inline manifest.json:29]: Did you confirm that we're ok to use this service? i.e. does Google's licence permit redistribution usage?

Comment (iTweeno) [reply manifest.json]: I asked David if this is fine, and he said it should be..

**Review by AshleyDawson — APPROVED** [review]

**Pull request merged**